File: langhuan-service/src/main/java/com/langhuan/controller/MockController.java
package com.langhuan.controller;


import com.langhuan.common.Result;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class MockController {

    @RequestMapping("/mock")
    public Result mock(
            @RequestBody Object data
    ) {
        return Result.success(data);
    }
}

File: langhuan-server/src/main/kotlin/com/langhuan/controller/MockController.kt
package com.langhuan.controller

import com.langhuan.common.Result
import org.slf4j.LoggerFactory
import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class MockController {

    companion object {
        private val log = LoggerFactory.getLogger(MockController::class.java)
    }

    @RequestMapping("/mock")
    fun mock(@RequestBody data: Any): Result<*> {
        return Result.success(data)
    }
}
